hq/app/views/fragments/openSecurityGroups.scala.html

```diff
@@ -73,6 +73,17 @@
                             <th>Region</th>
                             <td>@flaggedSg.region</td>
                         </tr>
+
+                        <tr>
+                            <th>Cloudformation Stack</th>
+                            <td>@flaggedSg.stackId match {
+                                case Some(id) => {
+                                    <a target="_blank" rel="noopener noreferrer" href="https://@{flaggedSg.region}.console.aws.amazon.com/cloudformation/home?region=@{flaggedSg.region}#/stack/detail?stackId=@helper.urlEncode(id)">@flaggedSg.stackName.get</a>
```

Comment:
`@flaggedSg.stackName.get`...
Is it possible to have a stackId, without a stackName? If so, is there a chance of trying to `get` a None here?

Reply:
`<a target="_blank" rel="noopener noreferrer"`
I would rather use target blank sparingly, and if we do, use it alongside the `open_in_new` md icon to indicate that we are overriding the default behaviour of the browser.

Not a blocker to getting this merged, however it would be good to overview/reduce the usage of target=_blank in the application. (Maybe the links could be target=_self then the icon could be target=_blank?)

Reply:
I think that's a great idea, pull that in as a consistent approach.

Reply:
@katebee I'm guessing it's unlikely to have stackId without a stack name, but i'll do a mini change for that.

Reply:
@katebee , This can be changed as below.

```
<td>@(flaggedSg.stackId, flaggedSg.stackName) match {
      case (Some(id), Some(name)) => {
        <a href="https://@{flaggedSg.region}.console.aws.amazon.com/cloudformation/home?region=@{flaggedSg.region}#/stack/detail?stackId=@helper.urlEncode(id)">@name</a>
      }
      case _ => {}
    }
</td>
```